guides/common/modules/proc_configuring-capsule-default-certificate.adoc

```diff
@@ -38,7 +38,6 @@ Retain a copy of the `{foreman-installer}` command that the `{certs-generate}` c
 _output omitted_
 {installer-scenario-smartproxy} \
 --certs-tar-file "/root/{smart-proxy-context}_cert/_{smartproxy-example-com}_-certs.tar" \
---foreman-proxy-register-in-foreman "true" \
```

Comment:
This is actually command output and comes from https://github.com/theforeman/foreman-installer/blob/2ec4fbdac78e208dbe83dc9414f73ab0dd60ee69/katello_certs/hooks/boot/02-message-helpers.rb#L60 so I think this particular line needs to be kept in until we drop it from the installer.